### PR TITLE
LocalBearTestHelper.py: Adds doc_string for execute_bear

### PR DESCRIPTION
**Pull request by manu-chroma**
Adds missing doc_string for execute_bear

Closes https://github.com/coala/coala-bears/issues/321

Includes improvements based on feedback from https://github.com/coala/coala-bears/pull/547

### CODE REVIEW

File: tests/LocalBearTestHelper.py [hunk 1]
```diff
@@ -14,6 +14,13 @@
 
 @contextmanager
 def execute_bear(bear, *args, **kwargs):
+    """
+    Execute the bear routine.
```
Comment (adtac) [inline tests/LocalBearTestHelper.py]: We usually leave a newline between the function doc and the param descriptions. I'm not sure if it's for some parser, but I'd suggest the same here.

Comment (manu-chroma) [inline tests/LocalBearTestHelper.py]: Done!

File: tests/LocalBearTestHelper.py [hunk 2]
```diff
@@ -14,6 +14,14 @@
...
+
```
Comment (manu-chroma) [inline tests/LocalBearTestHelper.py]: sorry. I added an extra file in first push, fixed that and then an ``autopep8`` warning showed up after pushing again.

Comment (manu-chroma) [inline tests/LocalBearTestHelper.py]: @adtac anything else I need to fix in this PR ?

**Comment by adtac** [comment]
ack https://github.com/coala/coala-bears/pull/883/commits/7b7e9c0664da7c19827f29acca02db39e67dd361

**Comment by adtac** [comment]
@rultor merge

File: tests/LocalBearTestHelper.py (hunk 2 above)
Comment (adtac) [inline tests/LocalBearTestHelper.py]: acked and started merge :+1:

Comment (manu-chroma) [inline tests/LocalBearTestHelper.py]: cool! 
also, quick question, what does ack stands for ?

**Comment by sils** [comment]
acknowledgement -> this commit is good to go

**Pull request merged**